README.md

```diff
@@ -1,79 +1,55 @@
-# The IOTA Bee framework
+<h1 align="center">
+  <br><img src="bee.png"></a>
+</h1>
 
-Bee is a Rust framework to build IOTA nodes and tools that interact with the IOTA network. The final product is
-envisioned to be a highly modular collection of foundational crates that can be mixed and matched, and extended upon.
+<h2 align="center">A Rust framework for building IOTA nodes and Tangle utilities after Coordicide</h2>
```

Comment:
I'd remove `after Coordicide`, this is currently not true

Reply:
The sentence  from the repo is `A framework for IOTA nodes, clients and applications in Rust`